.gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+# /node_modules
+node_modules/
+# /.pnp
+.pnp
```

Comment:
What are these pnp files?

Reply:
Not sure for .gitignore file I was using a template.

Reply:
https://datatypes.net/open-pnp-files